test/index.js

```diff
@@ -261,45 +253,52 @@ function runBench(options) {
   }
   bench('marked (pedantic)', marked);
 
-  // robotskirt
-  try {
-    bench('robotskirt', (function() {
-      var rs = require('robotskirt');
-      return function(text) {
-        var parser = rs.Markdown.std();
-        return parser.render(text);
-      };
-    })());
-  } catch (e) {
-    console.log('Could not bench robotskirt.');
-  }
-
   // showdown
   try {
     bench('showdown (reuse converter)', (function() {
       var Showdown = require('showdown');
-      var convert = new Showdown.converter();
+      var convert = new Showdown.Converter();
       return function(text) {
         return convert.makeHtml(text);
       };
     })());
     bench('showdown (new converter)', (function() {
       var Showdown = require('showdown');
       return function(text) {
-        var convert = new Showdown.converter();
+        var convert = new Showdown.Converter();
         return convert.makeHtml(text);
       };
     })());
   } catch (e) {
     console.log('Could not bench showdown.');
   }
 
+  // markdown-it
+  try {
+    bench('markdown-it', (function() {
+      var MarkdownIt = require('markdown-it');
+      var md = new MarkdownIt();
+      return function(text) {
+        return md.render(text);
+      };
+    })());
+  } catch (e) {
+    console.log('Could not bench markdown-it.');
```

Comment:
Here we can show error like this: `console.log('Could not bench markdown-it. (Error: %s)', e);`

Reply:
I was thinking of doing something like that.

Would we want to also check for `e.code === 'MODULE_NOT_FOUND'` and display a message like `Please install x to run bench`?

Reply:
I think it's better to make it simpler - just error message.

Reply:
done

Reply:
@UziTech: Maybe create an issue for the idea of "module not found" - I like the idea of decoupling us from packages and giving users a message like that. I'd flag it for 0.6.0 milestone as it improves the developer experience.

Reply:
The way I set it up now if they don't have a module installed they will see:

```
Could not bench showdown. (Error: Cannot find module 'showdown')
```